.travis.yml

```diff
@@ -117,9 +117,10 @@ deploy:
       all_branches: true
       condition: "$HAS_RELEASE_TAG == true"
   - provider: s3
-    access_key_id: AKIAQ6FOLK5RDONGHVBY
+    access_key_id:
+      secure: "QHenZfIwQHlgIAj0MVHmBk9hfAF5cYzvl9z2WiA2SC9YBbpfc5coyVeHwviE3gqb2kPSKujSZA7/GexmJuWvJMrOuuVTPtK9vJualXxntFK6pj3wLnxbsa3KEZu7ntcbuQwRy3i4QelTs5JswHJLLNsz/yh6JnI74j0DGNUu/oIBD/ZiWEqxQXwxme0/6MLZAidE4RvWvVfcME0JbIBCkUvUsJe3wZg/ftdGRyONImRn5w19hrkZHReFIA7uTjljRrxJlAvhJK1eKUJEEFe7N4LEzAYsO8i3yL5YhQWhuYYxDtOh+nq2Eb63VFMW0kFwSQB+Gu9YlA7x4vM+NxM6SRL2NdywILsmcoVpgJXXl4viOd3mIY1GK0ulafUx9Q3T+GTN/gJopziS3aTZntRq+6xdj5QR437f51W4Yjqjn8dBDopmv+jFx2flaHWPNcAee+YThK/p8WVqboRQqIDi50J8OslaDF1ALsygRQFEwPy6FK5U9z16KstN+DQ9QoAwBUIw9e5NWZE+pKA4SEsvlSFVGEeM+IiudaUoG/Bs4qgkmFpWvaCCk0Ki1dS4VdaTIp7M7hbY3Ad6Dy+aU279I5+iDUx1y2wMEknsbGzKNDFeimcdSxdLRElFoLP7l1UJjyH/wxazCxRfG8oCCQPFEivgLT7XYyf+4IHLnBhg3yw="
```

Comment:
In most examples I found, the access key id wasn't encrypted. However, I did just now find an example of it being encrypted too and I recall getting a warning from a random repo scanner (although it seemed to be phising) for not encrypting the access key id. I guess there's no harm in encrypting it just in case.